java/java.source.base/src/org/netbeans/modules/java/source/save/CasualDiff.java

```diff
@@ -1954,6 +1961,14 @@ protected int diffCase(JCCase oldT, JCCase newT, int[] bounds) {
             endpos = endPos(oldPatterns.get(oldPatterns.size() - 1));
 
             if (newPatterns.isEmpty()) {
+                if(copyTo > localPointer){
```

Comment:
Should this be something like `copyTo(localPointer, copyTo);`?

Reply:
Here I am trying to print the spaces before the default keyword as the formatting was wrong on using the hint. `copyTo(localPointer, copyTo) ` will copy the entire thing which could either be the case or default keyword.

Reply:
I guess you might need to use the lexer (there is a tokensequence in this class that can be used, IIRC) to find the end offset to copy. The code below will copy only spaces - but indent may be done using tabs, or even may contain comments. Finding the keyword using lexical tokens is probably easiest.

Reply:
Thanks for the input. Have made the changes.